tasks/buildah.yaml

```diff
@@ -63,6 +63,33 @@ spec:
     - mountPath: /var/lib/containers
       name: varlibcontainers
     workingDir: $(workspaces.source.path)
+  - image: quay.io/redhat-appstudio/syft:v0.46.2
+    name: sbom-source
+    args:
+      - dir:$(workspaces.source.path)
+      - --file=$(workspaces.source.path)/sbom-source.json
+      - --output=cyclonedx-json
+  - image: quay.io/redhat-appstudio/syft:v0.46.2
+    name: sbom-image
+    args:
+      - dir:/var/lib/containers
+      - --file=$(workspaces.source.path)/sbom-image.json
+      - --output=cyclonedx-json
+    volumeMounts:
+    - mountPath: /var/lib/containers
+      name: varlibcontainers
+  - image: $(params.BUILDER_IMAGE)
+    name: inject-sbom
+    resources: {}
+    script: |
+      container=$(buildah from --storage-driver=$(params.STORAGE_DRIVER) --pull-never $(params.IMAGE))
+      buildah --storage-driver=$(params.STORAGE_DRIVER) copy $container $(workspaces.source.path)/sbom-source.json /root/
```

Comment:
@stuartwdouglas - this is an example of the calling buildah from a script/shell in order to copy content out of an image in mentioned in the hacbs milestone 6 checkpoint today

in theory we could do something similar in the jvm-build-service for accessing content stored in an image (vs. S3 storage)

Reply:
thanks for the timely example @brunoapimentel :-)

Reply:
I should give the credits to @ejegrova, all I did was copying the code she wrote :D

Reply:
totally - and I did mention @ejegrova in our meeting this morning